Gulpfile.js

```diff
@@ -0,0 +1,48 @@
+const gulp = require('gulp');
+const fs = require('fs');
+const connect = require('gulp-connect');
+const jade = require('gulp-jade');
+const rename = require('gulp-rename');
+
+gulp.task('connect', () => {
+  connect.server({
+    root: 'build',
+    port: 8000,
+    livereload: true,
+  });
+});
+
+gulp.task('jade', () => {
+  gulp.src('jade/*.jade')
+    .pipe(jade({
+      locals: JSON.parse(fs.readFileSync('./data-source.json', 'utf8')),
```

Comment:
1) We can specify this as a command line option when you run `npm run gulp`!

2) If you put everything in a data directory, we can default all of those to be part of locals, with any optional parameters included.